pep_sphinx_extensions/pep_zero_generator/pep_index_generator.py

```diff
@@ -41,7 +59,7 @@ def create_pep_zero(_: Sphinx, env: BuildEnvironment, docnames: list[str]) -> No
     pep_pat = re.compile(r"pep-\d{4}")  # Path.match() doesn't support regular expressions
 
     # AUTHOR_OVERRIDES.csv is an exception file for PEP0 name parsing
-    with open("AUTHOR_OVERRIDES.csv", "r", encoding="utf-8") as f:
+    with open("AUTHOR_OVERRIDES.csv", encoding="utf-8") as f:
```

Comment:
Minor: I know it isn't required, but I find the explicit `"r"` helpful as a quick orientation device.

```suggestion
    with open("AUTHOR_OVERRIDES.csv", "r", encoding="utf-8") as f:
```

Reply:
I have no strong preference here; to avoid nitpicking and back and forth churn, a good strategy for us all to adopt, IMO (and for these sorts of nits in general) is just leaving existing instances as they are (with or without an implicit `"r"`) and following the your own preference for your own new/rewritten code (this is basically how we de-facto handle things like AmE/BrE spelling differences already).

Reply:
Okay, will ditch 6f07df99bf0167b3cda6c98b748a18fa6e39d893.